src/main/kotlin/racing/model/Car.kt

```diff
@@ -1,16 +1,28 @@
 package racing.model
 
-class Car {
-    private var moving: Int = 0
-    fun run(): Int {
-        val randomNumber: Int = (RANDOM_MIN_NUMBER..RANDOM_MAX_NUMBER).random()
-        if (randomNumber >= LIMIT_ADVANCE) moving++
-        return moving
+class Car(val name: String) {
+    companion object {
+        const val CAR_SYMBOL = "-"
+        private const val RANDOM_MIN_NUMBER = 0
+        private const val RANDOM_MAX_NUMBER = 9
+        private const val LIMIT_ADVANCE = 4
+        private const val MIN_NAME_LENGTH = 1
+        private const val MAX_NAME_LENGTH = 5
     }
 
-    companion object {
-        private const val RANDOM_MIN_NUMBER: Int = 0
-        private const val RANDOM_MAX_NUMBER: Int = 9
-        private const val LIMIT_ADVANCE: Int = 4
+    init {
+        require(name.length in MIN_NAME_LENGTH..MAX_NAME_LENGTH) {
+            "레이싱카 이름은 1글자 이상 5글자 이하입니다."
```

Comment:
사소할 수 있지만, 이렇게 예외메시지도 상수를 활용한다면 혹시라도 최소길이, 최대길이 제약조건이 변경되더라도 상수만 변경하면 되겠죠? 😃
```suggestion
            "레이싱카 이름은 $MIN_NAME_LENGTH글자 이상 $MAX_NAME_LENGTH글자 이하입니다."
```